src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -493,3 +502,18 @@
       return(list(error = e))
    })
 })
+
+.rs.addFunction("preview_data_import_async", function(jsonData)
+{
+   tryCatch({
+      params = jsonlite::fromJSON(jsonData)
```

Comment:
It'd be nice to avoid the jsonlite dependency -- we already have a set of internal methods that convert R objects to and from JSON (necessary for marshaling in RPCs). See `RJson.cpp`. Will that do what we need or is jsonlite doing something special here?